spec/features/media_viewer_spec.rb

```diff
@@ -71,7 +71,7 @@
       expect(page).to have_css('[data-slider-object="1"] video', visible: false)
 
       within('.sul-embed-thumb-slider') do
-        expect(page).to have_css('.sul-embed-slider-thumb', count: 2)
+        expect(page).to have_css('.sul-embed-slider-thumb', count: 3)
```

Comment:
ditto to comment below - this was a side effect of changing that fixture object.  Note that the number is increased by 1, not dropping, so nothing is being removed from thumb slider -- the additional file in the fixture is being added to the thumb slider.